test/functional/personal_access_token_test.rb

```diff
@@ -0,0 +1,95 @@
+require File.join(File.dirname(__FILE__), 'test_helper')
+
+describe 'personal_access_token' do
+  let(:base_cmd) { ['user', 'access-token'] }
+  let(:user) do
+    {
+     :id => 1,
+     :name => 'admin'
+    }
+  end
+  let(:access_token) do
+    {
+      :id => 1,
+      :user_id => user[:id],
+      :name => 'test',
+      :expires_at => '01/01/2048'
+    }
+  end
+
+  describe 'list' do
+    let(:cmd) { base_cmd << 'list' }
+    let(:params) { ['--user-id=1'] }
+
+    it 'lists all access tokens for a given user' do
+      api_expects(:personal_access_tokens, :index, 'List').with_params(
+        'user_id' => 1, 'page' => 1, 'per_page' => 1000
+      ).returns(index_response([access_token]))
+
+      expected_result = success_result(/#{access_token[:name]}/)
```

Comment:
There's a lot more to test then just name. Could you test all the columns, please?